next.config.js

```diff
@@ -7,9 +7,9 @@ const nextConfig = {
   // https://reactjs.org/docs/strict-mode.html
   reactStrictMode: true,
 
-  // https://nextjs.org/docs/messages/webpack5
-  future: {
-    webpack5: true,
+  // The pre-commit hook takes care of linting.
+  eslint: {
+    ignoreDuringBuilds: true,
```

Comment:
yeah, I think that's a good idea
sometimes style errors still made it through, and I think I'd rather not have that break the build